src/diffusers/models/unet_2d_condition.py

```diff
@@ -26,7 +26,10 @@
 from .embeddings import (
```

Comment:
Changes here look all good to me! would just be nice / important to add some tests and docstrings